foundations/introduction/motivation_and_mindset.md

```diff
@@ -179,3 +178,5 @@ This section contains helpful links to related content. It isn’t required, so
   - [TypingClub.com](https://www.typingclub.com)
   - [Monkeytype](https://monkeytype.com/)
 - [Why Procrastinators Procrastinate](https://waitbutwhy.com/2013/10/why-procrastinators-procrastinate.html) Learn about the Instant Gratification Monkey, Rational Decision Maker, Panic Monster, and how to navigate the Dark Playground. [Short video introduction from the same author](https://youtu.be/arj7oStGLkU)
+- The Ruby Rogues have a [podcast on How to Learn](https://topenddevs.com/podcasts/ruby-rogues/episodes/131-rr-how-to-learn), which should be motivational and useful to you, so check it out for some useful thoughts on learning.
```

Comment:
I've added the changes you've mentioned but I think the video is quite lengthy, would it be better to put it into additional resources instead? Feel like point 2 of assignment already goes quite in-depth into the growth mindset

Reply:
Thanks @nikitarevenco, 

> would it be better to put it into additional resources instead? Feel like point 2 of assignment already goes quite in-depth into the growth mindset. I want people to get coding faster!

I think its still worth having. It doesn't cover the same mindset stuff the second assignment does, its more about tactics while learning. It should compliment the growth/grit assignment well.

> Also, I think the 16-hour course may be a bit too much and I would consider that a rabbit hole honestly. Maybe its worth to remove the mention of that course and just have the video in the additional resources?

I agree, it would be too much for an assignment, but should be ok in the additional resources. I'd be reluctant to remove it entirely, its well regarded within the community. A lot of people thank us for introducing it to them. If it's in additional resources, its more of a choice than an obligation.

Reply:
> Thanks @nikitarevenco,
> 
> > would it be better to put it into additional resources instead? Feel like point 2 of assignment already goes quite in-depth into the growth mindset. I want people to get coding faster!
> 
> I think its still worth having. It doesn't cover the same mindset stuff the second assignment does, its more about tactics while learning. It should compliment the growth/grit assignment well.
> 
> > Also, I think the 16-hour course may be a bit too much and I would consider that a rabbit hole honestly. Maybe its worth to remove the mention of that course and just have the video in the additional resources?
> 
> I agree, it would be too much for an assignment, but should be ok in the additional resources. I'd be reluctant to remove it entirely, its well regarded within the community. A lot of people thank us for introducing it to them. If it's in additional resources, its more of a choice than an obligation.

Alright, I've added the mention of the course to additional resources